README.md

```diff
@@ -14,14 +14,21 @@ Hardwarové požadavky se odvíjí od počtu kontrol a přístupu uživatelů. P
 
 ## Spuštění aplikace
 
-Po naklonování repozitáře je potřeba zkopírovat lokální nastavení neonu a vyplnit údaje:
+1. Po naklonování repozitáře je potřeba zkopírovat lokální nastavení neonu a vyplnit údaje:
 
 ```
 cp config/config.local.example.neon app/config/config.local.neon 
 ```
+- Údaje pro `github appId, appSecret` můžete získat [zde](https://github.com/settings/applications/new),
+- do pole `Authorization callback URL` uveďte: `http(s)://vas-virtual.tld`. 
+- Pro získání `administratorTeamId` musíte disponovat [oAuth tokenem](https://github.com/settings/tokens) 
```

Comment:
"- Vyplněním `administratorTeamId` se budou uživatelé zařazení v danné skupině na GitHubu automaticky nastavovat jako administrátoři. Pro získání ..."